### PR TITLE
Replace rest-client with faraday

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

**Comment by jcoyne** [comment]
Looks like an engine_cart update is needed.

File: .gitignore [hunk 1]
```diff
@@ -17,4 +17,5 @@
 Gemfile.lock
 coverage
 pkg
-spec/internal
\ No newline at end of file
+.internal_test_app
+.internal_test_app
```
Comment (awead) [inline .gitignore]: duplicate line.

File: .travis.yml [hunk 2]
```diff
@@ -1,15 +1,15 @@
 language: ruby
-cache: bundler
+# cache: bundler
```
Comment (awead) [inline .travis.yml]: remove?

**Comment by awead** [comment]
:+1:

File: .travis.yml (hunk 2 above)
Comment (cbeer) [inline .travis.yml]: :ok_hand:

File: .gitignore (hunk 1 above)
Comment (cbeer) [inline .gitignore]: :ok_hand:

**Pull request merged**